e2e/pod.go

```diff
@@ -59,6 +59,13 @@ func waitForDeploymentComplete(name, ns string, c kubernetes.Interface, t int) e
 	err = wait.PollImmediate(poll, timeout, func() (bool, error) {
 		deployment, err = c.AppsV1().Deployments(ns).Get(context.TODO(), name, metav1.GetOptions{})
 		if err != nil {
+			if testutils.IsRetryableAPIError(err) {
+				// hide API-server timeouts, so that PollImmediate() retries
+				e2elog.Logf("deployment error: %v", err)
+				return false, nil
+			}
+			// FIXME: this is for debugging etcdserver timeouts only
+			e2elog.Logf("[DEBUG] deployment check failed: %v (%T)", err, err)
```

Comment:
remove Fixme?

Reply:
Well, it failed once, so we might still need to fix things here.

Reply:
And it actually failed the last run :partying_face: 

    Oct 27 11:27:25.860: INFO: [DEBUG] deployment check failed: etcdserver: request timed out (*errors.StatusError)

Reply:
This kind of error is now caught as well, and a retry will be done.